client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -113,15 +113,15 @@ export class DisplayTable extends React.Component {
         },
       }
       */,
+      page_size_increment,
```

Comment:
Small tangent from the specific cleanup feedback part of this, more of an API design point. Felt like a prop that let the consumer specify the page size was appropriate, but since the component itself allows users to change `page_size` this is instead more about setting the default page size and controlling the page size options available within the component.